jmx_prometheus_httpserver/src/main/java/io/prometheus/jmx/WebServer.java

```diff
@@ -18,6 +19,9 @@ public static void main(String[] args) throws Exception {
      Server server = new Server(port);
      ServletContextHandler context = new ServletContextHandler();
      context.setContextPath("/");
+     if (jc.gzipEnabled()) {
```

Comment:
not sure if we should.. i'd say keep it optional - and present it as a recommended setting on the docs.. WDYT?